core/templates/core/irap_request/steps/how_are_you_receiving_data.html

```diff
@@ -2,14 +2,21 @@
 
 {% load gds %}
 
-{% block page_title %}My page title{% endblock %}
-{% block page_heading %}My page heading{% endblock %}
+{% block page_title %}How are you receiving data{% endblock %}
+{% block page_heading %}How are you receiving the data?{% endblock %}
 
 {% block step_content %}
 <form method="post" novalidate>
     {% csrf_token %}
     {% include 'gds/error-summary.html' %}
+    <h2 class="govuk-heading-m">Select all that apply</h2>
+    {% gds_field form.receiving_data_methods template='gds/checkboxes.html' hide_label=True conditionals=True divider='or' %}
```

Comment:
The conditionals and divider are not required.
```suggestion
    {% gds_field form.receiving_data_methods template='gds/checkboxes.html' hide_label=True %}
```